src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -142,6 +151,19 @@ module.exports = React.createClass({
             );
         }
 
+        if (this.props.mxEvent.getType() === 'm.room.message') {
+            const content = this.props.mxEvent.getContent();
+            if (content.msgtype // truthy check msgtype
+            && content.msgtype !== 'm.bad.encrypted'
```

Comment:
this indentation feels very wrong - please can you indent the operators here?

```
if (foo
    && bar
    && baz
) {
    ...
}
```

is probably the right way to do it?